packages/api/tests/api/users.test.ts

```diff
@@ -12,9 +12,82 @@ const sampleUser: Partial<User> = {
   githubId: '234234',
 };
 
+const sampleSignedInUser: Express.User = {
+  profile: {
+    id: 'aaa',
+  },
+  githubToken: 'abcd123',
+};
+
 const loggerSpy = jest.spyOn(logger, 'error').mockImplementation();
 
-describe('users API GET route', () => {
+describe('GET users/me', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+  });
+
+  it('401 error when not logged in', async () => {
+    const handler = testHandler(users);
+
+    const { text } = await handler.get('/me').expect(401);
+    expect(text).toEqual('You must be logged in.');
+    expect(handler.entityManager.findOne).toBeCalledTimes(0);
+  });
+
+  it('404 error current user not found in database', async () => {
+    const handler = testHandler(users, (req, _res, next) => {
+      req.user = sampleSignedInUser;
+      next();
+    });
+
+    handler.entityManager.findOne.mockResolvedValueOnce(null);
+
+    await handler.get('/me').expect(404);
+    expect(handler.entityManager.findOne).toBeCalledTimes(1);
+    expect(handler.entityManager.findOne).toHaveBeenCalledWith(User, {
+      githubId: sampleSignedInUser.profile.id,
+    });
+  });
+
+  it('500 error during findOne', async () => {
+    const handler = testHandler(users, (req, _res, next) => {
+      req.user = sampleSignedInUser;
+      next();
+    });
+
+    handler.entityManager.findOne.mockRejectedValueOnce(new Error());
+
+    const { text } = await handler.get('/me').expect(500);
+
+    expect(loggerSpy).toBeCalledTimes(1);
+    expect(handler.entityManager.findOne).toBeCalledTimes(1);
+    expect(handler.entityManager.findOne).toHaveBeenCalledWith(User, {
+      githubId: sampleSignedInUser.profile.id,
+    });
+    expect(text).toEqual('There was an issue getting the currently logged in user');
+  });
+
+  it("Successfully send current user's information", async () => {
```

Comment:
Also try not to use apostrophes in the description; it makes Jest very, _**very**_ unhappy sometimes
```suggestion
  it("successfully sends the data for the user when the user is logged in", async () => {
```